app/templates/admin.hbs

```diff
@@ -1,5 +1,5 @@
 <div class="row">
-  <div class="sixteen wide column">
+  <div class="x-scrollable">
```

Comment:
@harshitagupta30 `sixteen wide column` is required here please do not simply replace it. All you got to do here is to add the `x-scrollable` class.